addons/event/models/event_event.py

```diff
@@ -480,7 +480,7 @@ def name_get(self):
             date_end = fields.Datetime.from_string(event.date_end)
             dates = [fields.Date.to_string(fields.Datetime.context_timestamp(event, dt)) for dt in [date_begin, date_end] if dt]
             dates = sorted(set(dates))
-            result.append((event.id, '%s (%s)' % (event.name, ' - '.join(dates))))
+            result.append((event.id, event.name))
```

Comment:
While you should triple check the effect of this change because most of place make assumption than date is into the event name... 
If you keep this idea, maybe you can remove also all the stuff that compute dates and the depends... if it is no more used.

Remark if you really doesn't want it for one specific event, you can just change the slug manually on the record.

Reply:
Hello,

@JKE-be Thank you for your advice :) I checked the routes and I do not see where we retrieve the dates straight from the url. I think that the system fetches the event from the database by using the provided id. Every computation will then be performed on that record. If a controller needs the dates of the event, it will simply look up in the fetched record. Or maybe I missed something... Maybe there is an obscure function that does a pattern matching on the url to avoid pinging the database. If so, can you provide me an example from the source code?

@awa-odoo I will change the commit message because it will indeed change the `_display_name` attribute.

Note: I've noticed that I can remove L479 to L481 as I no longer use these variables.

Reply:
@julien-banken 
No, controllers only use ID you are right.
I mean that it could impact the SEO where the date was expected (Even if only start date make sense for FP:
E.g.: odoo-experience-October-6-2847 instead of odoo-experience-2021-2021-10-06-2021-10-07-2847) or some forms view where it could be necessary to distinct the event. But it is a detail/arbitrary, probably you should ignore it... 

The main goal of my message was:

> If you keep this idea, maybe you can remove also all the stuff that compute dates and the depends... if it is no more used.

Yes, as you said in note, + remove dates from the depends on top of the function ;)